<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsCalloutPanelWidgetBase</class>
 <widget class="QgsPanelWidget" name="QgsCalloutPanelWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Callout</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Style</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="mCalloutStyleComboBox"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QStackedWidget" name="mCalloutStackedWidget">
     <widget class="QWidget" name="pageDummy">
      <layout class="QVBoxLayout" name="verticalLayout_19">
       <item>
        <widget class="QLabel" name="label_45">
         <property name="text">
          <string>This callout type doesn't have any editable properties</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsPanelWidget</class>
   <extends>QWidget</extends>
   <header>qgspanelwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
